src/App.tsx

```diff
@@ -1,185 +1,283 @@
-      onClickContinue={() => ending ? show('end') : show('event')}
-      onClickExtra={() => show('feedbackExtra')}
-      onClickSource={showSource}
-    />;
-    case 'feedbackExtra': return <FeedbackExtra
-      response={getLastResponse()}
-      onClickBack={() => show('feedback2')}
-      onClickSource={showSource}
-    />;
+    case "feedback1":
+      return (
+        <FeedbackScreen1
+          response={getLastResponse()}
+          feedback={getSocialFeedback(getLastResponse().socialMediaResponse)}
+          onClick={() => show("feedback2")}
+        />
+      );
+    case "feedback2":
+      return (
+        <FeedbackScreen2
+          response={getLastResponse()}
+          indicatorsLastTurn={getIndicatorsLastMonth()}
+          onClickContinue={() => (ending ? show("end") : show("event"))}
+          onClickExtra={() => show("feedbackExtra")}
+          onClickSource={showSource}
+        />
+      );
+    case "feedbackExtra":
+      return (
+        <FeedbackExtra
+          response={getLastResponse()}
+          onClickBack={() => show("feedback2")}
+          onClickSource={showSource}
+        />
+      );
 
     // End show
-    case 'end': return <Ending
-      leaderStyle={endings[ending]}
-      onClick={() => { show('allEndings') }}
-    />;
-    case 'allEndings': return <AllEndings
-      onClick={(name: string) => { setEnding(name); show('viewEnding') }}
-      onReplay={reset}
-    />;
-    case 'viewEnding': return <ViewEnding
-      leaderStyle={endings[ending]}
-      onClick={() => { show('allEndings') }}
-    />;
-    default: return <></>
+    case "end":
+      return (
+        <Ending
+          leaderStyle={endings[ending]}
+          onClick={() => {
+            show("allEndings");
+          }}
+        />
+      );
+    case "allEndings":
+      return (
+        <AllEndings
+          onClick={(name: string) => {
+            setEnding(name);
+            show("viewEnding");
+          }}
+          onReplay={reset}
+        />
+      );
+    case "viewEnding":
+      return (
+        <ViewEnding
+          leaderStyle={endings[ending]}
+          onClick={() => {
+            show("allEndings");
+          }}
+        />
+      );
+    default:
+      return <></>;
   }
-}
-
-const App = () => {
+};
 
+const App: React.FC = () => {
   const [fontReady, setFontReady] = useState(false);
 
   useEffect(() => {
-    const font = new FontFaceObserver('Bebas Neue')
-    font.load().then(() => {
-      setFontReady(true);
-    }).catch((err: Error) => {
-      console.error("FONT NOT READY")
-      console.error(err)
-      setFontReady(true)
-    })
-  })
-
-  return <>{fontReady && <GameLoop />}</>
-}
+    const font = new FontFaceObserver("Bebas Neue");
+    font
+      .load()
+      .then(() => {
+        setFontReady(true);
+      })
+      .catch((err: Error) => {
+        setFontReady(true);
+      });
+  });
+
+  return (
+    <>
+      <ToastContainer />
```

Comment:
This component can be used to position the toast notifications. It has good defaults so I left it as is, but it can be fully customized: https://fkhadra.github.io/react-toastify/introduction/